### PR TITLE
4732 feat cleanup the validate configuration modal

### LINKED ISSUE #4732: [FEAT]: Cleanup the validate configuration modal
**Describe the solution you'd like**
Styling and layout should look like the master designs:
Validate Model Configuration - Start

In the empty notebook tab, the display should match calibrate:
Screenshot 2024-09-09 at 3 53 24 PM

### PR DESCRIPTION
**Pull request by brittanykondo**
# Description

* Styling changes to the left side of the modal to look more like the designs (didn't spend much time on the constraint forms, since we're re-doing them)
* Made the left side collapsible

Screenshot 2024-09-12 at 12 19 28 PM

Resolves #4732

### CODE REVIEW

File: packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman.vue [hunk 1]
```diff
@@ -613,7 +597,45 @@ div.section-row.timespan > div > span {
...
+}
+
+:deep(.slider-tab) {
+	background-color: var(--surface-100);
+	border-right: 1px solid var(--surface-border-light);
+}
+
+:deep(.slider-tab header) {
+	background: transparent;
+}
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman.vue:633]: Would be better if you can just apply a class called `secondary` or `input-config` to the `tera-slider` and `tera-slider` would have the rules to handle that. 

Since I see this pattern used quite in other areas, this can also be cleaned up in another PR.

**Review by shawnyama — APPROVED** [review]

**Review by blanchco — APPROVED** [review]
looks fine, although i do see that transform-dataset.md change is still happening 👍
